packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -196,8 +196,7 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
               'interact with the VM service at runtime.\n'
               'This flag is ignored if "--start-paused" or coverage are requested, as '
               'the VM service will be enabled in those cases regardless.'
-      )
-      ..addOption('reporter',
+      )..addOption('reporter',
```

Comment:
I'd suggest reverting this formatting change so that this case continues to match with the others in this file.